src/Core/Models/Business/OrganizationLicense.cs

```diff
@@ -145,10 +147,10 @@ public OrganizationLicense()
     /// </summary>
     /// <remarks>Intentionally set one version behind to allow self hosted users some time to update before
```

Comment:
Is this comment still validate or we need to update it?

Reply:
This is still valid